nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -187,6 +187,7 @@ class Meta:
         """Meta attributes for SoftwareLCM."""
 
         verbose_name = "Software"
+        verbose_name_plural = "Software"
```

Comment:
I was testing this today and unfortunately this will break GraphQL type for the single `software`. The way we auto-generate GraphQL types relies on the `verbose_name` and `verbose_name_plural` `Meta` attributes.

https://github.com/nautobot/nautobot/blob/develop/nautobot/core/graphql/generators.py#L242-L243

We could change the `verbose_name_plural` to `Software List` but then we break the convention that is used by other types in GraphQL.

We might have to do some research here. The alternative is to inherit from the list view template and override the title block.

Reply:
Ah gotcha. Good call out. Let me revert those changes for now and we'll explore other options.